packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -468,16 +471,9 @@ export function getEditableTreeContext(forest: IEditableForest): EditableTreeCon
     return new ProxyContext(forest);
 }
 
-/**
- * Checks the type of an UnwrappedEditableField.
- */
-export function isArrayField(field: UnwrappedEditableField): field is UnwrappedEditableTree[] {
-    return Array.isArray(field);
-}
-
 /**
  * Checks the type of an UnwrappedEditableField.
  */
 export function isUnwrappedNode(field: UnwrappedEditableField): field is EditableTree {
-    return typeof field === "object" && !isArrayField(field);
+    return typeof field === "object" && field !== null && !Array.isArray(field);
```

Comment:
I don't think this change is needed: you didn't add any use of null did you?

Reply:
No, I didn't. Thought it might be better for completeness. Do you see any issues here (maybe, semantically)?

Reply:
`null` is not permitted by the `UnwrappedEditableField` type, so it would be invalid for a caller to pass in null here. Typescript treats null and undefined separately: handling null here is not needed.

If for some reason this function had a history of invalidly getting called with null and we wanted to help with that, an assert !null could be useful to help find such bugs and fix them, but I have no reason to think people will be violating the type system and calling this function with null.

Explicitly checking for, and handling (without throwing an error) malformed data that the type system forbids should be avoided: it just adds unnecessary code (with a download size and execution cost) and makes any bugs that pass such bad values less obviously bugs.